### PR TITLE
Merging in Upload Test Branch.

### PR DESCRIPTION
**Pull request by zaalb99**
- Added Resume upload functionality
- Added mailing and forget password functionality

### CODE REVIEW

**Comment by gperkins21** [comment]
Looks good! There are a couple of things I'll want to talk aobut in a call, but other than that it looks great!

File: gmail.env [hunk 1]
```diff
@@ -0,0 +1,2 @@
+GMAIL_ADDRESS=zaalb599@gmail.com
```
Comment (hagensopher) [inline gmail.env]: this env should not be here bad coding practice

**Review by hagensopher — COMMENTED** [review]
Read all the files just the env should be remvoed

**Comment by hagensopher** [comment]
Overall looks great

**Pull request merged**